### ISSUE TITLE
softstep 1 or 2

### ISSUE THREAD

**Issue by clfest** [X2theL/SoftStepQuark#1]
Is this code for softstep 1 or 2?

**Comment by beatboxchad** [comment]
After searching around today for a softstep 1 manual, not finding it, and reading the 2 one, I have come to the preliminary conclusion that as far as MIDI messages are concerned, they're the same, or largely similar. I'll come back and update if I find any differences.

**Comment by clfest** [comment]
Yes, they seem to be. The foot controllers work in a slightly different
way, so I managed to
hack the code to work with SS2. I have that code, and I will be happy to
share it, but it
needs some cleaning up to be of much use. Ideally one could create some
code that would
be of use for both pedals, the part where they differ being selectable by
the user.

**Comment by X2theL** [comment]
If you send me your code along with some good documentation of the differences I could give it a go. But I won't have time to do that until March. Maybe we could try this pull request thingy although,honestly, I have no idea how it works. I just hack things together as a hobby. But I can learn if you explain it to me :)
